test/transcoding/KernelArgTypeInOpString.ll

```diff
@@ -21,21 +21,31 @@
 ; OpString "kernel_arg_type.%kernel_name%.typename0,typename1,..."
 
 ; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv %t.bc -spirv-text -o %t.spv.txt
+; RUN: llvm-spirv -preserve-ocl-kernel-arg-type-metadata-through-string %t.bc -spirv-text -o %t.spv.txt
 ; RUN: FileCheck < %t.spv.txt %s --check-prefix=CHECK-SPIRV
```

Comment:
Could you rename this, to make it more apparent that it is checking for a workaround, please?

Reply:
I.e. rename CHECK-SPIRV to CHECK-SPIRV-WORKAROUND, or add a comment, or some equivalent that makes this clearer.

Reply:
Applied.
Basically the entire KernelArgTypeInOpString.ll and KernelArgTypeInOpString2.ll tests are for the workaround (note: current workaround with constant qualifier in kernel_arg_type_qual metadata is just an expansion for the one added years before, that adds translation of a kernel_arg_type(_name?) metadata). So there is already a comment in the test:  "As a workaround we store original names in OpString instruction".